server/app/utils.py

```diff
@@ -319,9 +319,13 @@ def add_to_grading_queues(assign_key, cursor=None, num_updated=0):
     seen = set()
     for queue in queues:
         for subm in queue.submissions:
-            seen.add(subm.get().submitter.id())
+            try:
```

Comment:
haha It's definitely possible to convert it; I can do it. On a side note, I thought Python was EAFP?